test/e2e/streaming.go

```diff
@@ -128,7 +128,7 @@ func startLongRunningContainer(rc internalapi.RuntimeService, ic internalapi.Ima
 	imageSpec := &kubeapi.ImageSpec{
 		Image: &latestTestImageRef,
 	}
-	err = ic.PullImage(imageSpec, nil)
+	_, err = ic.PullImage(imageSpec, nil)
 	framework.ExpectNoError(err, "Failed to pull image: %v", err)
```

Comment:
I'm afraid the streaming url has been changed on upstream: https://github.com/feiskyer/frakti/blob/60f69a711af0c98f378fe46a7fc535dd27c79fbd/test/e2e/streaming.go#L157